core/src/dird/vbackup.cc

```diff
@@ -77,7 +77,7 @@ class JobConsistencyChecker {
   std::vector<std::string> JobList;
   std::vector<std::string> JobsWithPurgedFiles;
 
-  bool operator()(int num_fields, char** row)
+  bool operator()([[maybe_unused]] int num_fields, char** row)
```

Comment:
That's not maybe unused, but definitely unused :)
```suggestion
  bool operator()(int, char** row)
```

Reply:
Is it not definitely used ?
```
assert(num_fields == 5);
```

I think the problem is that we build with NDEBUG set or something like that such that the assert is removed.  
Switching `assert` with `ASSERT` should probably work.